analyze.sh

```diff
@@ -14,7 +14,7 @@ if [ -d "$ARTIFACT_DIR" ]; then
   cp public/dist/report.html "${ARTIFACT_DIR}"
 fi
 
-MAX_BYTES=3827302 # ~3.65 MiB
+MAX_BYTES=3.879731 # ~3.7 MiB
```

Comment:
FYI @jhadvig, this is necessary since we're pulling a new dep in to the main vendor bundle.